sdk/src/types/block/output/feature/mod.rs

```diff
@@ -256,6 +257,19 @@ impl Features {
     }
 }
 
+impl WorkScore for Features {
+    fn work_score(&self, work_score_params: WorkScoreParameters) -> u32 {
```

Comment:
Tbh I think I would prefer doing it like Chloé did the StorageCost and how they do it in iota.go. Implement `WorkScore` for everything. It's clearer, less error-prone and overall I think it's nice to be able to have access to individual work scores.

Reply:
Done (initial implementation was like this btw)